Makefile

```diff
@@ -26,4 +26,21 @@ test:
 lint:
 	./hack/shellcheck.sh
 
+setup_env:
+	./scripts/feature_tests/setup_env.sh
+
+cleanup_env:
+	./scripts/feature_tests/cleanup_env.sh
+
+pivoting_test:
+	make -C ./scripts/feature_tests/pivoting/
+
+remediation_test:
+	make -C ./scripts/feature_tests/remediation/
+
+upgrade_test:
+	make -C ./scripts/feature_tests/upgrade/
+
+feature_tests: setup_env remediation_test cleanup_env upgrade_test cleanup_env pivoting_test
+
```

Comment:
This is ok for now, but IMO, the framework should rather be based on a script that iterates through the folders in scripts/feature_tests rather than having it hardcoded. And it could take a parameter to run the tests of the folder that matches the parameter only.

Reply:
Agree, but lat's do is in a separate PR.